spec/system/batch_process_spec.rb

```diff
@@ -48,9 +39,6 @@
         expect(BatchProcess.count).to eq 1
         expect(page).to have_content("Your records have been retrieved from the MetadataCloud. PTIFF generation, manifest generation and indexing happen in the background.")
         expect(BatchProcess.last.file_name).to eq "meta.xml"
-        within "td.count" do
-          expect(page).to have_content('1')
-        end
```

Comment:
Removed because the td classes no longer exist now that we're using a data table and not a bootstrap table. these values are being checked in `spec/datatables/batch_process_datatable_spec.rb`